aries-site/src/pages/index.js

```diff
@@ -25,11 +25,15 @@ const HomeTiles = ({ ...rest }) => {
 // Reasoning for using forwardRef: https://nextjs.org/docs/api-reference/next/link#example-with-reactforwardref
 const TopicTile = forwardRef(({ topic, ...rest }, ref) => {
   return (
+    // Needs to be <a> in DOM for web crawling: https://support.google.com/webmasters/answer/9112205?hl=en
     <Tile
+      as="a"
       pad="medium"
       background={topic.color}
       key={topic.color}
       ref={ref}
+      fill
+      style={{ textDecoration: 'none' }}
```

Comment:
I'd consider creating a new styled component of <AnchoredTile /> that will have a combination of lines 30, 35 and 36, to distinguish it from a regular tile, not a must but cleaner and more readable.

Reply:
Pulling over my reply on slack:
I'm open to that, I didn't want to refactor to another component if it didn't seem necessary, but if I were to, I could call it `AnchorTile`?

Reply:
`AnchorTile` sounds good to me.